### PR TITLE
Made Terminology Consistent

### PR DESCRIPTION
**Pull request by jaskiratsingh2000**
Made terminology case sensitive in the "Channels" folder

### CODE REVIEW

File: source/channels/channel-types.rst [hunk 1]
```diff
@@ -18,22 +18,22 @@ Channel types
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-You'll find four types of channels in Mattermost: public channels, private channels, direct messages, and group messages.
+You'll find four types of channels in Mattermost: Public Channels, Private Channels, direct messages, and group messages.
```
Comment (justinegeffen) [inline source/channels/channel-types.rst:21]:
```suggestion
You'll find four types of channels in Mattermost: public channels, private channels, direct messages, and group messages.
```

File: source/channels/channel-types.rst [hunk 2]
```diff
@@ -18,22 +18,22 @@ Channel types
...
 
 Public channels
 ---------------
 
-Public channels are open to everyone on a team and are identified with a |globe-icon| icon. New team members are automatically added to two public channels when they sign up: **Town Square** and **Off-Topic**. See the :doc:`Join and Leave Channels </channels/join-leave-channels>` documentation for details on discovering and joining other channels.
+Public channels are open to everyone on a team and are identified with a |globe-icon| icon. New team members are automatically added to two Public Channels when they sign up: **Town Square** and **Off-Topic**. See the :doc:`Join and Leave Channels </channels/join-leave-channels>` documentation for details on discovering and joining other channels.
```
Comment (justinegeffen) [inline source/channels/channel-types.rst:26]:
```suggestion
Public channels are open to everyone on a team and are identified with a |globe-icon| icon. New team members are automatically added to two public channels when they sign up: **Town Square** and **Off-Topic**. See the :doc:`Join and Leave Channels </channels/join-leave-channels>` documentation for details on discovering and joining other channels.
```

File: source/channels/channel-types.rst [hunk 3]
```diff
@@ -18,22 +18,22 @@ Channel types
...
 
 .. |globe-icon| image:: ../images/globe-icon.png
    :alt: A globe icon indicates a public channel.
 
 Private channels
 ----------------
 
-Private channels are for sensitive topics and are only visible to selected team members. Private channels are identified with a |lock-icon| icon. In Team Edition, any member of a private channel can add or remove other members from private channels, but in other Mattermost versions these permissions can be restricted to the Channel Admins and System Admins.
+Private channels are for sensitive topics and are only visible to selected team members. Private channels are identified with a |lock-icon| icon. In Team Edition, any member of a Private Channel can add or remove other members from Private channels, but in other Mattermost versions these permissions can be restricted to the Channel Admins and System Admins.
```
Comment (justinegeffen) [inline source/channels/channel-types.rst:34]:
```suggestion
Private channels are for sensitive topics and are only visible to selected team members. Private channels are identified with a |lock-icon| icon. In Team Edition, any member of a private channel can add or remove other members from private channels, but in other Mattermost versions these permissions can be restricted to the Channel Admins and System Admins.
```

File: source/channels/channel-types.rst [hunk 4]
```diff
@@ -18,22 +18,22 @@ Channel types
...
 
-Channel members can choose to leave private channels at any time.
+Channel members can choose to leave Private Channels at any time.
```
Comment (justinegeffen) [inline source/channels/channel-types.rst:36]:
```suggestion
Channel members can choose to leave private channels at any time.
```

File: source/channels/convert-public-channels.rst [hunk 5]
```diff
@@ -1,4 +1,4 @@
-Convert public channels to private channels
+Convert Public Channels to Private Channels
```
Comment (justinegeffen) [inline source/channels/convert-public-channels.rst:1]:
```suggestion
Convert public channels to private channels
```

File: source/channels/convert-public-channels.rst [hunk 6]
```diff
@@ -20,12 +20,12 @@ Convert public channels to private channels
 
 Team and System Admins can convert public channels to private channels. 
 
-Select the public channel name at the top of the center pane to access the drop-down menu, then select **Convert to Private Channel**. When a channel is converted, its history and membership are preserved. Membership in a private channel is by invitation only. Publicly-shared files remain accessible to anyone with the link. 
+Select the public channel name at the top of the center pane to access the drop-down menu, then select **Convert to private channel**. When a channel is converted, its history and membership are preserved. Membership in a private channel is by invitation only. Publicly-shared files remain accessible to anyone with the link.
```
Comment (justinegeffen) [inline source/channels/convert-public-channels.rst:23]:
```suggestion
Select the public channel name at the top of the center pane to access the drop-down menu, then select **Convert to Private Channel**. When a channel is converted, its history and membership are preserved. Membership in a private channel is by invitation only. Publicly-shared files remain accessible to anyone with the link.
```

File: source/channels/convert-public-channels.rst [hunk 7]
```diff
@@ -20,12 +20,12 @@ Convert public channels to private channels
...
 Team and System Admins can convert public channels to private channels. 
 
-Select the public channel name at the top of the center pane to access the drop-down menu, then select **Convert to Private Channel**. When a channel is converted, its history and membership are preserved. Membership in a private channel is by invitation only. Publicly-shared files remain accessible to anyone with the link. 
+Select the public channel name at the top of the center pane to access the drop-down menu, then select **Convert to private channel**. When a channel is converted, its history and membership are preserved. Membership in a private channel is by invitation only. Publicly-shared files remain accessible to anyone with the link. 
 
 .. note::
     Default channels such as ``Town Square`` and ``Off-Topic`` can't be converted to private channels.
 
-Convert private channels to public channels
+Convert Private Channels to Public Channels
```
Comment (justinegeffen) [inline source/channels/convert-public-channels.rst:28]:
```suggestion
Convert private channels to public channels
```

File: source/channels/create-channels.rst [hunk 8]
```diff
@@ -43,15 +43,15 @@ To create a public or private channel, select the **+** symbol at the top of the
 Create a direct or group message
 --------------------------------
 
-To start a direct or group message, select the **+** symbol next to the **Direct Messages** category in the channel sidebar.
+To start a direct or group message, select the **+** symbol next to the **direct messages** category in the channel sidebar.
```
Comment (justinegeffen) [inline source/channels/create-channels.rst:46]:
```suggestion
To start a direct or group message, select the **+** symbol next to the **Direct Messages** category in the channel sidebar.
```

File: source/channels/create-channels.rst [hunk 9]
```diff
@@ -43,15 +43,15 @@ To create a public or private channel, select the **+** symbol at the top of the
...
 
 .. image:: ../images/write-dm.png
-    :alt: Access recent Direct Messages and Group Messages.
+    :alt: Access recent direct messages and group messages.
 
 .. tip::
     Alternatively, you can select the **+** symbol at the top of the channel sidebar, then select **Open a Direct Message**. 
 
-In the **Direct Messages** popup, identify your most recent conversations by relative timestamps.     
+In the **direct messages** popup, identify your most recent conversations by relative timestamps.
```
Comment (justinegeffen) [inline source/channels/create-channels.rst:54]:
```suggestion
In the **Direct Messages** popup, identify your most recent conversations by relative timestamps.
```

File: source/channels/customize-your-channel-sidebar.rst [hunk 10]
```diff
@@ -23,7 +23,7 @@ Conversations in Mattermost are crucial to company productivity and success. Kee
 Here's how your sidebar is set up by default:
 
 - All public and private channels you've joined are listed in the **Channels** category, sorted alphabetically.
-- All your direct messages and group messages are listed in the **Direct Messages** category, sorted by recent activity.
+- All your direct messages and group messages are listed in the **direct messages** category, sorted by recent activity.
```
Comment (justinegeffen) [inline source/channels/customize-your-channel-sidebar.rst:26]:
```suggestion
- All your direct messages and group messages are listed in the **Direct Messages** category, sorted by recent activity.
```

File: source/channels/customize-your-channel-sidebar.rst [hunk 11]
```diff
@@ -162,13 +162,13 @@ To sort your direct messages, select the **Channel options** icon in the sidebar
 How many direct messages to display?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Control how many direct message conversations display in the **Direct Messages** category to keep your conversations manageable. You can choose to show all messages or a fixed number of messages.
+Control how many direct message conversations display in the **direct messages** category to keep your conversations manageable. You can choose to show all messages or a fixed number of messages.
```
Comment (justinegeffen) [inline source/channels/customize-your-channel-sidebar.rst:165]:
```suggestion
Control how many direct message conversations display in the **Direct Messages** category to keep your conversations manageable. You can choose to show all messages or a fixed number of messages.
```

File: source/channels/customize-your-channel-sidebar.rst [hunk 12]
```diff
@@ -162,13 +162,13 @@ To sort your direct messages, select the **Channel options** icon in the sidebar
...
 
 To configure the number of direct messages to display, go to **Settings > Sidebar**, then set **Number of direct messages to show**. Or select the **Channel options** icon in the sidebar, then select **Show**.
 
-Choose from **All direct messages**, or choose to show **10**, **15**, **20**, or **40** messages. Once you exceed the number of Direct Messages configured, older messages are hidden from the **Direct Messages** category. You can always increase the number of conversations displayed to see older Direct Messages.
+Choose from **All direct messages**, or choose to show **10**, **15**, **20**, or **40** messages. Once you exceed the number of Direct Messages configured, older messages are hidden from the **direct messages** category. You can always increase the number of conversations displayed to see older direct messages.
```
Comment (justinegeffen) [inline source/channels/customize-your-channel-sidebar.rst:169]:
```suggestion
Choose from **All direct messages**, or choose to show **10**, **15**, **20**, or **40** messages. Once you exceed the number of direct messages configured, older messages are hidden from the **Direct Messages** category. You can always increase the number of conversations displayed to see older direct messages.
```

File: source/channels/customize-your-channel-sidebar.rst [hunk 13]
```diff
@@ -162,13 +162,13 @@ To sort your direct messages, select the **Channel options** icon in the sidebar
...
 
 .. image:: ../images/dm-display.gif
 
 .. note::
-  Direct message conversations that you add to custom categories don't count against the maximum number of conversations shown in the **Direct Messages** category.
+  Direct message conversations that you add to custom categories don't count against the maximum number of conversations shown in the **direct messages** category.
```
Comment (justinegeffen) [inline source/channels/customize-your-channel-sidebar.rst:174]:
```suggestion
  Direct message conversations that you add to custom categories don't count against the maximum number of conversations shown in the **Direct Messages** category.
```

File: source/channels/keyboard-shortcuts-for-channels.rst [hunk 14]
```diff
@@ -28,7 +28,7 @@ The following keyboard shortcuts are supported in all `supported browsers <https
 +----------------------------+---------------------------+----------------------------------------------------------------------------------+
 | CTRL+K                     | CMD+K                     | Open a quick channel selector dialog.                                            |
 +----------------------------+---------------------------+----------------------------------------------------------------------------------+
-| CTRL+SHIFT+K               | CMD+SHIFT+K               | Open the **Direct Messages** dialog.                                             |
+| CTRL+SHIFT+K               | CMD+SHIFT+K               | Open the **direct messages** dialog.                                             |
```
Comment (justinegeffen) [inline source/channels/keyboard-shortcuts-for-channels.rst:31]:
```suggestion
| CTRL+SHIFT+K               | CMD+SHIFT+K               | Open the **Drect Messages** dialog.                                             |
```

File: source/channels/react-to-messages.rst [hunk 15]
```diff
@@ -62,7 +62,7 @@ Everyone in your Mattermost workspace can use the custom emojis you upload. From
   
 .. note::
 
-  If you can't see the **Custom Emoji** option, then your Mattermost System Admin may have restricted access to upload custom emoji. Contact your Mattermost System Admin for assistance.
+  If you can't see the **custom emoji** option, then your Mattermost System Admin may have restricted access to upload custom emoji. Contact your Mattermost System Admin for assistance.
```
Comment (justinegeffen) [inline source/channels/react-to-messages.rst:65]:
```suggestion
  If you can't see the **Custom Emoji** option, then your Mattermost System Admin may have restricted access to upload custom emoji. Contact your Mattermost System Admin for assistance.
```

File: source/channels/react-to-messages.rst [hunk 16]
```diff
@@ -72,10 +72,10 @@ Remove custom emojis
 You can remove custom emojis that you uploaded to Mattermost.
   
 1. Open the Emoji Picker.
-2. Select **Custom Emoji**.
-3. If required, use the Search Bar to find your Custom Emoji in the list.
+2. Select **custom emoji**.
```
Comment (justinegeffen) [inline source/channels/react-to-messages.rst:75]:
```suggestion
2. Select **Custom Emoji**.
```

File: source/channels/search-for-messages.rst [hunk 17]
```diff
@@ -174,7 +174,7 @@ Searching Chinese, Korean, and Japanese
 Differences between PostgreSQL and MySQL search
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, Mattermost uses full text search support included in MySQL and PostgreSQL. These databases have slightly different search behavior. Check **Product menu > About Mattermost** to see which database you’re using.
+By default, Mattermost uses full text search support included in MySQL and PostgreSQL. These databases have slightly different search behavior. Check **product menu > About Mattermost** to see which database you’re using.
```
Comment (justinegeffen) [inline source/channels/search-for-messages.rst:177]:
```suggestion
By default, Mattermost uses full text search support included in MySQL and PostgreSQL. These databases have slightly different search behavior. Check **Product menu > About Mattermost** to see which database you’re using.
```

File: source/channels/set-channel-preferences.rst [hunk 18]
```diff
@@ -57,7 +57,7 @@ To stop receiving mention notifications, enable the **Ignore mentions for @chann
 Send desktop notifications
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default, your `desktop notification preferences <https://docs.mattermost.com/channels/channels-settings.html#notifications>`__ configured in **Settings** apply to all channels. Desktop notifications are available on Edge, Firefox, Safari, Chrome, and `Mattermost Desktop Apps <https://mattermost.com/download/#mattermostApps>`__.
+By default, your `desktop notification preferences <https://docs.mattermost.com/channels/channels-settings.html#notifications>`__ configured in **Settings** apply to all channels. Desktop notifications are available on Edge, Firefox, Safari, Chrome, and `Mattermost desktop Apps <https://mattermost.com/download/#mattermostApps>`__.
```
Comment (justinegeffen) [inline source/channels/set-channel-preferences.rst:60]:
```suggestion
By default, your `desktop notification preferences <https://docs.mattermost.com/channels/channels-settings.html#notifications>`__ configured in **Settings** apply to all channels. Desktop notifications are available on Edge, Firefox, Safari, Chrome, and `Mattermost desktop apps <https://mattermost.com/download/#mattermostApps>`__.
```

**Review by justinegeffen — COMMENTED** [review]
Thanks for this great PR, @jaskiratsingh2000! It's really solid work and I appreciate your attention to detail.

I have provided suggestions and, where applicable, corrections inline. Please let me know if anything is unclear!

Thank you! :)

**Comment by jaskiratsingh2000** [comment]
Hi @justinegeffen I have updated the commits according to the changes mentioned

**Review by justinegeffen — APPROVED** [review]
Great work! Thank you! :)

**Pull request merged**